web/apps/labelstudio/src/pages/ExportPage/ExportPage.jsx

```diff
@@ -235,7 +235,13 @@ const FormatInfo = ({ availableFormats, selected, onClick }) => {
       <Elem name="feedback">
         Can't find an export format?
         <br/>
-        Please let us know in <a className="no-go" href="https://slack.labelstud.io/?source=product-export">Slack</a> or submit an issue to the <a className="no-go" href="https://github.com/heartexlabs/label-studio-converter/issues">Repository</a>
+        Please let us know in
+        {" "}
+        <a className="no-go" href="https://slack.labelstud.io/?source=product-export" target="_blank">Slack</a>
+        {" "}
+        or submit an issue to the
+        {" "}
```

Comment:
this looks like we're using spaces to adjust positioning/spacing - im guessing the styles applied to the `<a>` is not showing the right amount of space (probably no spacing at all)
this should be addressed in the the style sheet

Reply:
it's not about adding paddings, this way we add usual spaces trimmed by react

Reply:
looking into this, its not so much that spaces around tags are removed as much as it is removing spaces at the beginning and end of the line - I dont particularly like this approach but having this paragraph on one line isnt much better - I can see personal preference coming into play here so i'll go ahead and resolve the convo/approve the pr

Reply:
sorry to correct you, but leading/trailing spaces are stripped only around tags, if you have usual multilined text, lines will be separated by spaces in final text. but not if line starts or ends with tag.